skimage/color/colorconv.py

```diff
@@ -1021,13 +1021,15 @@ def xyz2lab(xyz, illuminant="D65", observer="2", *, channel_axis=-1):
 
 @channel_as_last_axis()
 def lab2xyz(lab, illuminant="D65", observer="2", *, channel_axis=-1):
-    """CIE-LAB to XYZcolor space conversion.
+    """Convert image in CIE-LAB to XYZ color space.
 
     Parameters
     ----------
     lab : (..., 3, ...) array_like
-        The image in Lab format. By default, the final dimension denotes
-        channels.
+        The input image in CIE-LAB color space.
+        By default, the final dimension denotes channels.
+        The range of L* values is 0 to 100;
+        the range of a* values and b* values is -128 to 127.
```

Comment:
```suggestion
        The range of L* is 0 to 100;
        the range of a* and b* is -128 to 127.
```

Reply:
Hmm, I guess it would still be understandable (unambiguous), but it's standard and commonplace to say either "the value range" or "the range of values," isn't it? So I would prefer keeping "values" here... Actually, I copied-pasted you from https://stackoverflow.com/a/46418321 :wink:

Reply:
English is weird :) My preferences (i.e., what sounds best to my ear):

1. the range of L* is 0 to 100
2. L* ranges from 0 to 100
3. The L* values range from 0 to 100
4. the range of the L* values is 0 to 100

Reply:
:upside_down_face: Do you mean, your preferences... *in decreasing order*?

(the preference rank ranges from 1 to 4) :rofl:

Reply:
I went for 3. (92ecb97e0b4c75a2c62fabe2d0eea6336f2760e1) because it sounds like a good compromise: It says "values" and I like how explicit it is with "*from* ... *to* ..."
I'm sparing you one instance of values though (with "a* and b* values")! :heart:

Reply:
I'll take it!